scripts/core/INTRO.rst

```diff
@@ -302,3 +302,16 @@ Specific environment variables can be set to control the behavior of unified run
    .. note::
 
     This environment variable should be used together with :envvar:`UR_ENABLE_VALIDATION_LAYER` and :envvar:`UR_LOG_VALIDATION`.
+
+Service identifiers
+---------------------
+
+The Unified Runtime API can log unique device identifiers.
+For security and functionality reasons, the user can enable and disable the logging of identifiers
```

Comment:
This is off by default. So instead, I think we can simply say:
`Unified Runtime may create logs containing Personally Identifiable Information (PII) in the form of unique device identifiers during its use. This capability is turned off by default. Please refer to the Logging section above for more information.`

Or maybe this should just be a sentence in the logging section?

Reply:
Done.